apps/transport/test/transport/jobs/consolidate_bnlc_job_test.exs

```diff
@@ -542,48 +542,12 @@ defmodule Transport.Test.Transport.Jobs.ConsolidateBNLCJobTest do
         end
       )
 
-      Transport.ExAWS.Mock
-      |> expect(:request!, fn %ExAws.Operation.S3{} = operation ->
-        assert %ExAws.Operation.S3{
-                 bucket: "transport-data-gouv-fr-on-demand-validation-test",
-                 path: path,
-                 http_method: :put,
-                 service: :s3
-               } = operation
-
-        assert path =~ ~r"^bnlc-.*\.csv$"
-      end)
-
-      Transport.EmailSender.Mock
-      |> expect(:send_mail, fn "transport.data.gouv.fr" = _display_name,
-                               "contact@transport.data.gouv.fr" = _from,
-                               "deploiement@transport.data.gouv.fr" = _to,
-                               "contact@transport.data.gouv.fr" = _reply_to,
-                               "[OK] Rapport de consolidation de la BNLC" = _subject,
-                               "",
-                               html_part ->
-        assert html_part =~ ~r"^✅ La consolidation s'est déroulée sans erreurs"
-
-        # Make sure a link is there
-        assert html_part =~
-                 ~r{🔗 <a href="https://transport-data-gouv-fr-on-demand-validation-test.cellar-c2.services.clever-cloud.com/bnlc-.*\.csv">Fichier consolidé</a>}
-
-        :ok
-      end)
+      expect_s3_upload()
```

Comment:
Mémoire de poisson rouge alors que j’ai tout reviewé , je me rappelle plus pourquoi on devait faire un appel à s3… J’avais l’impression qu’on faisait tout en local sur le /tmp du worker, puis (nouvelle partie) on uploade à la fin sur datagouv.

Ça me fait dire qu’il est peut-être temps de remplacer le « Need to write some documentation » en expliquant le workflow : téléchargement des resources séparées depuis la liste des resources, check du schéma, etc, etc (et où et comment techniquement).